cmd/k8s-bigip-ctlr/main_test.go

```diff
@@ -324,6 +314,43 @@ var _ = Describe("Main Tests", func() {
 			Expect(err).ToNot(BeNil())
 		})
 
+		It("gets credentials from a file", func() {
```

Comment:
We should add an additional test that verifies credentials-directory values supersede the other CLI arguments. Each one should be provided via both methods and make sure that the ones set come from the k8s secrets.